product_docs/docs/pgd/5/manual/03-configuring-repositories.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Step 3 - Configuring PGD repositories
+navTitle: Configuring PGD repositories
+deepToC: true
+---
+
+## Configuring PGD repositories
+
+To install and run PGD requires that you configure repositories so that the system can download the appropriate packages.
+
+The following operations should be carried out on each host. For the purposes of this exercise, each host will be a standard data node, but the procedure would be the same for other node types such as witness or subscriber-only nodes.
+
+Obtain your EDB repository token from the [EDB Repos 2.0](https://www.enterprisedb.com/repos-downloads) page.
+
+Set it as en environment variable
+
+Run the automated installer to install the repositories:
+
+!!! Note Red Hat
+```
+curl -1sLf "https://downloads.enterprisedb.com/$EDB_SUBSCRIPTION_TOKEN/postgres_distributed/setup.rpm.sh" | sudo -E bash
+```
+!!!
+
+!!! Note Ubuntu/Debian
+```
+curl -1sLf "https://downloads.enterprisedb.com/$EDB_SUBSCRIPTION_TOKEN/postgres_distributed/setup.deb.sh" | sudo -E bash
+```
+!!!
+
+## Worked example
+
+### EDB account
+
+You'll need an EDB account to install Postgres Distributed. 
+
+You should have an EDB account. Use this to sign in to the [EDB Repos 2.0](https://www.enterprisedb.com/repos-downloads) page, where you can obtain your repo token. 
+
+On your first visit to this page, select **Request Access** to generate your repo token. Copy the token using the **Copy Token** icon, and store it safely. 
+
+### Setting environment variables
```

Comment:
Ah, here it is. This part should be moved upwards or needs to be duplicated.

Reply:
Duplicated into the general guide.